pubspec.lock

```diff
@@ -1899,4 +1899,4 @@ packages:
     version: "3.1.2"
 sdks:
   dart: ">=3.2.0 <3.13.0"
-  flutter: ">=3.16.0"
+  flutter: ">=3.16.0"
```

Comment:
Why is this a change ? Did you add or remove the EOF line ?

Reply:
No I do not remove or add the EOF line

Reply:
Maybe it is because I have taken the lock file from develop branch and pasted in my branch

Reply:
You can try merging your branch with develop. Remove the pubspec in a commit, merge with develop, and the do the final push. That should do. Also, it's a nice strategy to keep merging with develop to avoid these conflicts here.

Reply:
ok